backend/Tasque.BLL/JWT/JwtFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tasque.Core.BLL.JWT
+{
+    public class JwtFactory
+    {
+        public readonly IConfiguration _config;
+
+        public JwtFactory(IConfiguration config)
+        {
+            _config = config;
+        }
```

Comment:
I don't think that you can get IConfiguration from DI container. imo it's better to have ConfigJwtFactory method that takes IConfiguration and hold jwtOptions and securityKey as static fields

Reply:
You can)